<launch>

    <!-- TODO: launch lidar node-->

    <node name="predictive_proxemics_navigation" pkg="predictive_proxemics_navigation" type="predictive_proxemics_navigation_node" output="screen">
        <param name="logFilePath" value="/home/nes/thesis_simon/logs/log_r_01.csv"/>
        <param name="robotStartX" value="-5.0"/>
        <param name="robotStartY" value="0.0"/>
        <param name="robotGoalX" value="9.0"/>
        <param name="robotGoalY" value="-0.74"/>
        <!-- <param name="robotGoalX" value="0.0"/> -->
        <!-- <param name="robotGoalY" value="0.0"/> -->
    </node>
</launch>